package.json

```diff
@@ -8,8 +8,10 @@
     "test": "test"
   },
   "scripts": {
-    "test": "test",
+    "test": "npm run check && npm run cover:unit",
```

Comment:
I think it's better to not run the check before tests. I don't want to format code every time I run tests locally. It makes sense to format code right before you commit.

Reply:
do you mean separated steps on ci ?

Reply:
https://github.com/commercetools/commercetools-subscriptions/pull/3/commits/b8aa33c21ddd392e74706d3931c894d916bb237a

Reply:
Yes